packages/backend/src/payment-method/ilp/peer/service.test.ts

```diff
@@ -427,8 +427,9 @@ describe('Peer Service', (): void => {
   describe('Peer pagination', (): void => {
     getPageTests({
       createModel: () => createPeer(deps, { assetId: asset.id }),
-      getPage: (pagination: Pagination | undefined) =>
-        peerService.getPage(pagination)
+      getPage: (pagination?: Pagination, sortOrder?: SortOrder) =>
+        peerService.getPage(pagination, sortOrder),
+      sortOrder: Math.random() < 0.5 ? SortOrder.Asc : SortOrder.Desc
```

Comment:
up to you, but we could actually put this line in `baseModel.test.ts` so we don't have to add this for every `getPageTests` separately